chalicelib/mapping.json

```diff
@@ -0,0 +1,172 @@
+{
+  "mappings": {
+      "result": {
+        "properties": {
+          "action": {
+            "type": "text",
+            "fields": {
+              "keyword": {
+                "type": "keyword",
+                "ignore_above": 256
+              }
+            }
+          },
+          "action_message": {
+            "type": "text",
+            "fields": {
+              "keyword": {
+                "type": "keyword",
+                "ignore_above": 256
+              }
+            }
+          },
+          "allow_action": {
+            "type": "boolean"
+          },
+          "brief_output": {
+            "enabled": false,
+            "type": "object",
+            "include_in_all": false
+          },
+          "description": {
+            "type": "text",
+            "fields": {
+              "keyword": {
+                "type": "keyword",
+                "ignore_above": 256
+              }
+            }
+          },
+          "ff_link": {
+            "type": "text",
+            "fields": {
+              "keyword": {
+                "type": "keyword",
+                "ignore_above": 256
+              }
+            }
+          },
+          "full_output": {
+            "enabled": false,
+            "type": "object",
+            "include_in_all": false
+          },
+          "kwargs": {
```

Comment:
Ignoring `kwargs` in the mapping may be preferable. That said, there are certain ubiquitous kwargs that could be useful to search by (such as "primary"). We could experiment with removing any kwargs that aren't standard; that is, create mappings for some of them and then, since dynamic mapping is turned off, newly encountered ones wouldn't be mapped. We'd have to test to see if this works